docs/release-notes/breaking-change-policy.asciidoc

```diff
@@ -0,0 +1,26 @@
+[[breaking-changes-policy]]
+=== Breaking changes policy
+
+
+The {java-client} source code is generated from a https://github.com/elastic/elasticsearch-specification[formal specification of the Elasticsearch API]. This API specification is large, and although it is tested against hundreds of Elasticsearch test files, it may have discrepancies with the actual API that result in issues in the {java-client}.
+
+Fixing these discrepancies in the API specification result in code changes in the {java-client}, and some of these changes can require code updates in your applications.
+
+This section explains how these breaking changes are considered for inclusion in {java-client} releases.
+
```

Comment:
```suggestion
[discrete]
==== Elasticsearch Stability Guarantees

All Elasticsearch APIs have stability indicators, which imply potential changes. If an API is `stable` only additional non breaking changes are added. In case of `experimental` APIs, breaking changes can be introduced any time, which means that these changes, will also be reflected in the {java-client}. 
```